Frameworks/CoreGraphics/CGImage.mm

```diff
@@ -346,8 +527,7 @@ CGImageRef CGImageCreateCopyWithColorSpace(CGImageRef ref, CGColorSpaceRef color
  @Status Stub
  */
 CGImageRef CGImageCreateWithMask(CGImageRef image, CGImageRef mask) {
-    // TODO #1124: Given how masks are applied during rendering via D2D, we will hold onto the
-    // mask then apply it at the appropriate time.
+    // TODO #1124
```

Comment:
Got it.

---
In reply to: [88774090](https://github.com/Microsoft/WinObjC/pull/1430#discussion_r88774090) [](ancestors = 88774090)